js/components/portal-dashboard/feedback/feedback-question-rows.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Map } from "immutable";
+import Answer from "../../../containers/portal-dashboard/answer";
+import striptags from "striptags";
+import { renderHTML } from "../../../util/render-html";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  answers: any;
+  currentActivity: Map<string, any>;
+  currentStudentId: string | null;
+  feedbacks: Map<any, any>;
+  feedbacksNeedingReview: Map<any, any>;
```

Comment:
I think I commented on this in another module.  Is this for future use?  I don't see it used anywhere.